source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,51 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+An alternative to DataTranslators is to use :javadoc:DataFormat, which allows you to store a ``DataContainer`` in HOCON, JSON or NBT format. You can also recreate DataConainters using ``DataFormats``.
+
+For example, you could use ``DataFormat`` to create a json repusentation of a ``DataContainer``, which you could then easily store in a database as JSON. ``DataFormat`` can then be used to recreate the original ``DataContainer`` from json where required.
+
+For this example we will use :javadoc:`DataFormats#JSON` to translate a ItemStackSnapshot to JSON and back! 
+
+**Imports for code examples**
+    import org.spongepowered.api.Sponge;
+    import org.spongepowered.api.data.DataContainer;
+    import org.spongepowered.api.data.DataQuery;
+    import org.spongepowered.api.data.DataView;
+    import org.spongepowered.api.data.persistence.DataBuilder;
+    import org.spongepowered.api.data.persistence.DataFormat;
+    import org.spongepowered.api.data.persistence.DataFormats;
+    import org.spongepowered.api.item.inventory.ItemStackSnapshot;
+
+    import java.io.ByteArrayOutputStream;
+    import java.io.IOException;
+    import java.util.Optional;
+
+**Code Example: Serializing a ItemStackSnapshot to JSON format**
+    
+.. code-block:: java
+
+    public ByteArrayOutputStream toJsonFormat(ItemStackSnapshot itemStack){
+        DataContainer playerDataContainer = itemStack.toContainer();
+        DataFormat dataFormat = DataFormats.JSON;
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        try {
+            dataFormat.writeTo(outputStream, playerDataContainer);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return outputStream;
+    }
+
+**Code Example: Deserializing a ItemStackSnapshot from json format**
```

Comment:
So I know it's been discussed before that you want to show how to use the `DataContainer` resulting from being converted from `JSON`, but I feel that it should still be shown *how* to convert it from JSON to a DataContainer, @dualspiral, I'm open for suggestions. I don't want to make it back and forth, but the wording is a little awkward since you state here that it's deserializing from JSON format, but there's no mention of JSON.